docs/content/src/learn/guides/services/converting-json-to-xml-and-upload-to-ftp/README.md

```diff
@@ -0,0 +1,157 @@
+# Convert JSON to XML and upload to FTP
+
+## About 
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the support of connectors. In this guide, we are mainly focusing on connecting to a file server with the FTP connector. You can find other integrations modules from the [wso2-ballerina](https://github.com/wso2-ballerina) Github repository.
+
+The FTP connector support as listener where listen to given remote location and trigger when file added or deleted. Also, it supports to connect to a file server as the client and perform various operations such as get, put, delete, etc. The FTP connector can be used to connect to your remote file server with FTP, SFTP, FTPS protocols.
+
+## What you'll build
+The Ballerina has first-class support for HTTP and implementing an HTTP service is straightforward. The caller will send JSON payload that consists of employees data. The HTTP service read the JSON payload and transform to XML. Then with the support of FTP connector, the XML employees data upload to the FTP server.
+
+## Prerequisites
+Link to download ballerina integrator.
```

Comment:
Can we add some more prerequisites similar to the other guide? Things like Java, IDE, FTP server, etc.

Reply:
done.